### ISSUE TITLE
Genes to Phenotype is down at compbio.charite.de

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2377]
http://compbio.charite.de/jenkins/job/hpo.annotations/lastSuccessfulBuild/artifact/util/annotation/genes_to_phenotype.txt

Hopefully temporary. Investigate if the resource is not online soon. It has an impact on how we map genes to phenotypes in Scout.

**Comment by dnil** [comment]
Hm, didn't they write something about permanently switching to the jax alias url?

**Comment by dnil** [comment]
Yes:

>Please note that due to unforeseen circumstances the URLs where HPOAs were previously made available (compbio.charite.de/jenkins) will not be working anymore soon and we have moved everything to servers from the monarch-initiative. Please always use the PURLs (permanent URLs) for HPO and HPOAs, which we list here: https://hpo.jax.org/app/download/annotation

**Comment by northwestwitch** [comment]
When?

**Comment by northwestwitch** [comment]
But didn't we fix it? I mean the test should still be patching the right function, but didn't we change that link?

**Comment by dnil** [comment]
Hm, tentatively seems we fixed one of three?

```
HPO_URL = (
    "http://compbio.charite.de/jenkins/job/hpo.annotations/lastStableBuild/"
    "artifact/util/annotation/{0}"
)
HPO_URL = (
    "http://compbio.charite.de/jenkins/job/hpo.annotations/lastSuccessfulBuild"
    "/artifact/util/annotation/{}"
)
HPOTERMS_URL = "http://purl.obolibrary.org/obo/hp.obo"
```

**Comment by dnil** [comment]
Or one of two, rather.. 🤦

**Comment by dnil** [comment]
HPO_URL = "http://purl.obolibrary.org/obo/hp/hpoa/{}" should do the trick.

**Comment by northwestwitch** [comment]
Fixed in #2378
